packages/components/src/pages/quickstart-docs/OpenAppMaps.vue

```diff
@@ -61,6 +81,21 @@ export default {
   },
 
   methods: {
+    goToInstallAgent(event) {
```

Comment:
@dustinbyrne I am adding click events to the three blue circles at the bottom of the view so that the user can navigate back and forth in the flow. I think I got it correct, but I can't test it in StoryBook. Does this look right to you?